### DISCUSSION TITLE
Fooocus installation for 3d meshes and lower PCs

### DISCUSSION THREAD

**Discussion by Dralokonda** [lllyasviel/Fooocus#3664] [Q&A] [answered]
I recently came into contact with a associate who highly recommended me to learn AI stable diffusion, in particularly Fooocus. The issue is my laptop is a Dell Vostro Intel Core with a intel xe graphics card, 134217728 kolobytes, 4 cores and 8 logical processors with 8 gb ram. 

He also recommended to me to find a way to incorporate Stable Diffusion into my 3D modeling Workflow. By taking my base meshes in autodesk maya and using stable diffusion to texture and light the model. 

I'd like some assistance in knowing how best to install stable diffusion and fooocus on my pc aswell as what prompts i can use to install stable diffusion in autodesk maya. Also, i have a maya 2020 and need to know whether i should update it to a newer version.

**Comment by IPv6** [comment]
Seems your device below minimal requirements - https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement

with your specific requirements - maya, etc - you may have better luck working with comfy, fooocus is not for direct integration with other software. there is web interface and that is all

**Reply by Dralokonda** [reply]
Ok, do you know how i can integrate it with maya, is there a prompt i must download, or can i export the unwrapped uvs of the mesh into comfy

**Reply by IPv6** [reply, marked as answer]
There is no such instructions for Fooocus. You run Fooocus (read https://github.com/lllyasviel/Fooocus/ for install etc) via web/browser.

With comfy there are a lot of tools regarding 3D, but this is open source, be ready to try, cry and experiment a lot

**Reply by Dralokonda** [reply]
ok, but is it comfyui or just comfy

**Reply by IPv6** [reply]
i was refering comfyui. most opensource integrations (Krita, Blender) have some comfyui-based addons. So there should be something similar for maya too, imho

**Reply by Dralokonda** [reply]
ok

**Reply by Dralokonda** [reply]
Btw, do i need to install python on my pc before installing comfyui, or is python part of the package
